src/CoamClient.js

```diff
@@ -405,8 +405,10 @@ class CoamClient {
         return this.__exec(data);
     }
 
-    getUserPermissionsForResourceType(principal, resourceType) {
-        let url = this.__buildUrl(`/auth/access-management/v1/principals/{{principal}}/permissions/{{resourceType}}`, {
+    getUserPermissionsForResourceType(principal, resourceType, include = false, permissionFilters = null) {
+        let urlPath = `/auth/access-management/v1/principals/{{principal}}/permissions/{{resourceType}}`;
+        urlPath = include && permissionFilters ? urlPath + `?include=true&permissionFilter={{permissionFilters}}`: urlPath;
```

Comment:
`include` and `permissionFilters` are independent parameter. Let's use them separately.